extensions/cornerstone/src/hooks/useViewportSegmentations.ts

```diff
@@ -0,0 +1,221 @@
+import { useState, useEffect } from 'react';
+import debounce from 'lodash.debounce';
+import { roundNumber } from '@ohif/core/src/utils';
+import {
+  SegmentationData,
+  SegmentationRepresentation,
+} from '../services/SegmentationService/SegmentationService';
+import { useSystem } from '@ohif/core';
+const excludedModalities = ['SM', 'OT', 'DOC', 'ECG'];
```

Comment:
Would prefer to use displaySet.images to test if this is an image containing modality.  That way video is excluded even if it is another modality.

Reply:
but RTSS does not have images, and yet still it is included. Am i missing something?